test/e2e/e2e.yaml

```diff
@@ -0,0 +1,117 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+setup:
+  env: kind
+  file: kind.yaml
+  steps:
+    - name: install cert-manager
+      command: |
+        # kind k8s cluster is in $TMPDIR
+        mkdir -p ~/.kube
+        cp $TMPDIR/e2e-k8s.config ~/.kube/config
```

Comment:
It's better not to override the users' Kubernetes config file, just set the config context via env var or in command line option

Reply:
ok, get it